_posts/en/newsletters/2020-05-06-newsletter.md

```diff
@@ -0,0 +1,241 @@
+  presented in virtual reality at Reckless VR. Osuntokun presented Tor
+  and RPC enhancements in the latest release of LND plus a new channel
+  feature called [anchor outputs][topic anchor outputs] that addresses
+  the challenge of estimating onchain fees months in advance.
+  Jager discussed the challenges of [multipart payments][topic multipath
+  payments] including the splitting algorithm, what happens when the
+  shards of the payment arrive at different times, and strategies for
+  when a multipart payment fails. Gugger ended by discussing Partially
+  Signed Bitcoin Transaction ([PSBT][topic psbt]) channel funding and
+  the channel abstraction work that made this possible.
+  ([transcript][lnd10 xs], [video][lnd10 vid])
+
+- **Grokking Bitcoin** Kalle Rosenbaum both participated in a [Socratic
+  Seminar][] and gave a presentation at London Bitcoin Devs.  The
+  seminar focused on the role of Bitcoin technical
+  education, [BIP32][] HD wallets, and soft fork upgrades.  For the
+  presentation, Rosenbaum used the content in his book to discuss how
+  the segwit upgrade of 2017 addressed transaction malleability and
+  quadratic hashing.  ([Socratic transcript][grok xs2], [presentation
+  transcript][grok xs1], [presentation video][grok vid], [presentation
+  slides][grok slides])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.8.2][c-lightning 0.8.2] is a new version release
+  that adds support for opening channels of any size (using the
+  `--large-channels` configuration parameter), provides a keysend plugin
+  for receiving [spontaneous payments][topic spontaneous payments], and
+  contains several other new features and bug fixes. Additionally, both
+  novice and experienced users are encouraged to read the project's new
+  [FAQ][cl faq].
+
+- [LND 0.10.0-beta][lnd 0.10.0-beta] is a major version release that
+  adds support for sending [multipath payments][topic multipath
+  payments], funding channels using an external wallet via [Partially
+  Signed Bitcoin Transactions][topic psbt] (PSBTs), the ability to create
+  invoices larger than 0.043 BTC, and several other new features and bug
+  fixes.  Additionally, users may wish to read the new [Operational
+  Safety documentation][lnd op safety].
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] is a release candidate
+  for the next major version of Bitcoin Core.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
```

Comment:
This boilerplate warning was added last week as well. What's the justification for doing this? Doesn't the same warning apply to commits for other repos?

Reply:
> This boilerplate warning was added last week as well.

Variations on the notice also appear in newsletters 9, 10, 11, 12, 13, 14, 37, 40, 41, 42, 43, 67, and 68.  (Search for "master" in any of those newsletters.)

> What's the justification for doing this?

Without the warning, I think people will expect the changes we describe to appear in the next Bitcoin Core release (e.g. 0.20 in a few weeks).

> Doesn't the same warning apply to commits for other repos?

No.  Both LND and C-Lightning RCs releases are just tags on their master branch.  I think that's also the case for Eclair, but I'm not sure.  Rust-Lightning has tags that GitHub shows as releases but there's no release notes and they're all version 0.0.x, so I've been assuming they're not real releases.  The other repos don't do releases.

In short, AFAICT, Bitcoin Core is the only project we monitor that branches from master prior to a major release.  (Indeed, I'm unable to think of any other free software project I've followed that does that.)

As an alternative to the warning, we could simply not report on Bitcoin Core commits to master during RC cycles, though that would leave us with a huge backlog when at the end of each cycle.

Reply:
ah ok. Makes sense. Thanks!